Src/AutoFixture/Dsl/CompositePostprocessComposer.cs

```diff
@@ -10,6 +10,8 @@ namespace AutoFixture.Dsl
     /// Aggregates an arbitrary number of <see cref="IPostprocessComposer{T}"/> instances.
     /// </summary>
     /// <typeparam name="T">The type of specimen to customize.</typeparam>
+    [Obsolete("This class is deprecated and will be removed in future versions of the product. " +
```

Comment:
"of AutoFixture" I'd rather say instead "of the product"

Reply:
Done, thanks for advise 😉